HW1/P2/P2b.py

```diff
@@ -1,3 +1,38 @@
 from P2 import *
+from pyspark import SparkContext
+import random
 
-# Your code here
+if __name__ == "__main__":
+    # initialize SparkContext
+    sc = SparkContext(appName="Mandelbrot P2b")
+    
+    # create a vector from 0-1999 (2000 elements) across 10 partitions
+    vector = sc.parallelize(range(0,2000),10)
```

Comment:
You can use sc.range instead of creating the range in python and moving it to spark.